runtime/common/src/weights/pallet_parachain_staking.rs

```diff
@@ -32,694 +33,663 @@
-	/// Storage: ParachainStaking Total (r:1 w:1)
-	/// Proof Skipped: ParachainStaking Total (max_values: Some(1), max_size: None, mode: Measured)
-	/// Storage: ParachainStaking BottomDelegations (r:1 w:1)
-	/// Proof Skipped: ParachainStaking BottomDelegations (max_values: None, max_size: None, mode: Measured)
+	/// Storage: `System::Account` (r:349 w:349)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(116), added: 2591, mode: `MaxEncodedLen`)
+	/// Storage: `ParachainStaking::DelegatorState` (r:349 w:349)
+	/// Proof: `ParachainStaking::DelegatorState` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainStaking::DelegationScheduledRequests` (r:1 w:0)
+	/// Proof: `ParachainStaking::DelegationScheduledRequests` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Balances::Locks` (r:349 w:349)
+	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1287), added: 3762, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Freezes` (r:349 w:0)
+	/// Proof: `Balances::Freezes` (`max_values`: None, `max_size`: Some(37), added: 2512, mode: `MaxEncodedLen`)
+	/// Storage: `ParachainStaking::CandidateInfo` (r:1 w:1)
+	/// Proof: `ParachainStaking::CandidateInfo` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainStaking::TopDelegations` (r:1 w:1)
+	/// Proof: `ParachainStaking::TopDelegations` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainStaking::CandidatePool` (r:1 w:1)
+	/// Proof: `ParachainStaking::CandidatePool` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainStaking::Total` (r:1 w:1)
+	/// Proof: `ParachainStaking::Total` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainStaking::BottomDelegations` (r:1 w:1)
+	/// Proof: `ParachainStaking::BottomDelegations` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// The range of component `x` is `[0, 349]`.
 	/// The range of component `y` is `[0, 349]`.
 	/// The range of component `z` is `[0, 349]`.
 	fn pay_one_collator_reward_best(x: u32, y: u32, z: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `0 + x * (395 ±0) + y * (156 ±0) + z * (41 ±0)`
-		//  Estimated: `125757 + x * (2591 ±1) + y * (2234 ±1) + z * (28 ±0)`
-		// Minimum execution time: 84_000 picoseconds.
-		Weight::from_parts(89_000, 0)
-			.saturating_add(Weight::from_parts(0, 125757))
-			// Standard Error: 586_928
-			.saturating_add(Weight::from_parts(37_021_750, 0).saturating_mul(x.into()))
-			// Standard Error: 586_928
-			.saturating_add(Weight::from_parts(20_024_306, 0).saturating_mul(y.into()))
+		//  Measured:  `0 + x * (396 ±0) + y * (156 ±0) + z * (41 ±0)`
+		//  Estimated: `126334 + x * (2591 ±0) + y * (2234 ±1) + z * (28 ±0)`
+		// Minimum execution time: 275_000 picoseconds.
+		Weight::from_parts(307_000, 126334)
```

Comment:
@librelois this call read cost increased more than tripled. Probably fine, but just wanted to mention it.

Reply:
I can't find any code change that explain this difference, but it doesn't impact that much in practice because this function can be executed at most once per block, so I'm fine with the change